modules/cluster-logging-updating-logging-to-5-0.adoc

```diff
@@ -97,7 +86,7 @@ elasticsearch-cdm-1pbrl44l-3-88df5d47-m45jc     2/2     Running   0          29m
 +
 [source,terminal]
 ----
-$ oc exec -n openshift-logging -c elasticsearch elasticsearch-cdm-1pbrl44l-1-55b7546f4c-mshhk -- es_cluster_health
+$ oc exec -n openshift-logging -c elasticsearch elasticsearch-cdm-1pbrl44l-1-55b7546f4c-mshhk -- health
```

Comment:
Down at line 98 - those three dots show up in red in the output.  I don't think they belong there?

Reply:
Removed. Thanks.